image.yaml

```diff
@@ -137,10 +137,10 @@ artifacts:
        md5: 3dba80cc1be17b00cb901441111886f3
        url: http://$DOWNLOAD_SERVER/released/JBoss-middleware/eap7/7.3.0/jboss-eap-7.3.0.zip
 
-     - name: jboss-eap-7.3.9-patch
-       target: jboss-eap-7.3.9-patch.zip
-       md5: 00779380dacdbfa01bfb3e62a307279a
-       url: http://$DOWNLOAD_SERVER/released/JBoss-middleware/eap7/7.3.9/jboss-eap-7.3.9-patch.zip
+     - name: jboss-eap-7.3.10-patch
+       target: jboss-eap-7.3.10-patch.zip
+       md5: 04b1dd4df9a979c47610be53a971992e
+       url: http://$DOWNLOAD_SERVER/devel/released/JBoss-middleware/eap7/7.3.10/jboss-eap-7.3.10-patch.zip
```

Comment:
@drichtarik The link can't have both `devel` & `released/`. Drop the `devel/` from it:

```
 curl -I http://$DOWNLOAD_SERVER/devel/released/JBoss-middleware/eap7/7.3.10/jboss-eap-7.3.10-patch.zip
HTTP/1.1 404 Not Found
Date: Mon, 17 Jan 2022 17:35:24 GMT
Server: Apache
Content-Type: text/html; charset=iso-8859-1
```
compared to (working one):
```
$ curl -I http://$DOWNLOAD_SERVER/released/JBoss-middleware/eap7/7.3.10/jboss-eap-7.3.10-patch.zip
HTTP/1.1 200 OK
Date: Mon, 17 Jan 2022 17:36:24 GMT
Server: Apache/2.4.6 (Red Hat Enterprise Linux) OpenSSL/1.0.2k-fips
Last-Modified: Wed, 15 Dec 2021 13:11:17 GMT
ETag: "9e1bb9c-5d32f0a94b4d4"
Accept-Ranges: bytes
Content-Length: 165788572
Content-Type: application/zip
```
The md5sum is OK.